lms/data_tasks/report/create_from_scratch/02_entities/01_users/01_raw_users/01_create_view.jinja2.sql

```diff
@@ -0,0 +1,15 @@
+DROP MATERIALIZED VIEW IF EXISTS report.raw_users CASCADE;
+
+-- The building blocks of a user. We won't export this. See
+-- `*_entities_decorated/*_users` for the version for external consumers.
+CREATE MATERIALIZED VIEW report.raw_users AS (
+    SELECT
+         id,
+         username,
+         registered_date::date
+    FROM h.user AS users
+    WHERE
+        users.authority = '{{ region.authority }}'
+        -- users.authority = 'lms.hypothes.is'
```

Comment:
Left from testing?

Reply:
I've left these in permanently as they are kind of handy when copy pasting things into Metabase to test